src/apps/inventory/migrations/0005_category_and_product_slug_populating.py

```diff
@@ -0,0 +1,48 @@
+# Generated by Django 4.2.5 on 2023-10-03 09:02
+
+from django.db import migrations, models
+
+
+def populate_slug(apps, schema_editor):
+    Product = apps.get_model('inventory', 'Product')
+    Category = apps.get_model('inventory', 'Category')
+
+    for product in Product.objects.all():
+        product.slug = f'{product.name}_{product.id}'
+        product.save()
+
+    for category in Category.objects.all():
+        category.slug = f'{category.name}_{category.id}'
+        category.save()
```

Comment:
you create a slug using name and id here but in save methods you still use only name

Reply:
fixed